src/developer/web-api/route.md

```diff
@@ -140,6 +130,22 @@ In the example shown below, we are configuring a route with `http-basic` authent
 
 Custom authorities allows a DHIS2 client that does not have the rights to manage the route to still be able to run it. This enables the route to be run from your app.
 
+### Running a route with custom response timeout
+
+The pre-configured response timeout for the origin when running a route is 5 seconds. A response taking longer than this will result in the DHIS2 server returning a gateway timeout error to the client. However, this timeout can be adjusted when creating or updating a route as demonstrated below:
+
+```json
+{
+  "name": "Postman Echo",
+  "code": "postman-post",
+  "disabled": false,
+  "url": "https://postman-echo.com/post",
+  "responseTimeoutSeconds": 10
+}
+```
+
+The minimum permitted response timeout is 1 second while the maximum permitted timeout is 60 seconds. The `responseTimeoutSeconds` setting should be used with caution since concurrent, long-running routes could degrade DHIS2's overall performance.
+
```

Comment:
We should add a warning block quote here about the fixed timeouts in older versions and the need to assign a custom timeout for long-running routes after upgrading to 42.  We should also add that to the upgrade notes here : https://github.com/dhis2/dhis2-releases/tree/master/releases/2.42